CONTRIBUTING.md

```diff
@@ -10,10 +10,17 @@ We actively welcome your pull requests.
 1. Fork the repo and create your branch from `master`.
 2. If you've added code that should be tested, add tests.
 3. If you've changed APIs, update the documentation.
-4. Ensure the test suite passes. Run `python setup.py test`. If you've added a new
-   dataset, you should run add a tests for it in `tests/data`.
-5. Make sure your code lints (`bash tests/lint_changed.sh`)
-6. If you haven't already, complete the Contributor License Agreement ("CLA").
+4. Make sure your code lints (`bash tests/lint_changed.sh`)
+5. Ensure the test suite passes. Run `python setup.py test`.
+6. If you've added a new dataset, you should also run
+   `python setup.py test -s tests.suites.datatests`. Copy-paste the output into a
+   comment in your PR.
+7. If you haven't already, complete the Contributor License Agreement ("CLA").
+
+Our continuous integration suite will run only fast tests before merging. We also
```

Comment:
Thanks this is exactly what i was referring to

Reply:
ya i went and updated it after you pointed it out :P